test/others/skip-file-rwx-check/tester.c

```diff
@@ -0,0 +1,10 @@
+#include <fcntl.h>
+#include <unistd.h>
+
+int main(int argc, char *argv[])
+{
+	open(argv[1], O_RDONLY | O_APPEND);
```

Comment:
need to check a return code and exit if open fails.

Reply:
Removed C code.